lib/appmap/handler/rails/request_handler.rb

```diff
@@ -46,8 +49,7 @@ def to_h
                     value: self.class.display_string(val),
                     object_id: val.__id__,
                   }.tap do |message|
-                    properties = object_properties(val)
-                    message[:properties] = properties if properties
+                    AppMap::Event::MethodEvent.add_schema message, val, always: true
```

Comment:
It appears this function is not available as of this commit. It's only introduced in the next one.

Reply:
I am doing my best to separate the commits but don't always nail it completely :-)